src/Misc/HighDynamicRange/cubemapToSphericalPolynomial.ts

```diff
@@ -136,7 +140,7 @@ export class CubeMapToSphericalPolynomialTools {
                                 fileFace.worldAxisForNormal);
                     worldDirection.normalize();
 
-                    var deltaSolidAngle = Math.pow(1.0 + u * u + v * v, -3.0 / 2.0);
+                    var deltaSolidAngle = AreaElement(u - 0.5 * du, v - 0.5 * dv) - AreaElement(u - 0.5 * du, v + 0.5 * dv) - AreaElement(u + 0.5 * du, v - 0.5 * dv) + AreaElement(u + 0.5 * du, v + 0.5 * dv);

```

Comment:
could we extract u-0.5du and such to reuse the result as they appear several times in here ?